### PR TITLE
Edited personal links and formatting

### PR DESCRIPTION
**Pull request by astromariarosa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
 # stateoftheuniverse
-Returns the state of the universe
-An Astro Hack Week 2019 project
+## A python package that returns the current state of the universe
 
+Would you like to know when the phase of the moon? What about the current tally of exoplanets and asteroids? Curious about the constellations and planets you can see in the sky tonight? Want to celebrate the birthdays of historical astronomers?
+
+Then you are going to love stateoftheuniverse.
+
+Input your location, and we'll do the rest.
```
Comment (ojhall94) [inline README.md:10]: can you add a little section on how to install? Just like
```
pip install stateoftheuniverse --user
```

Comment (ojhall94) [reply README.md:10]: Actually at the moment teach them how to download it and install, so

```
git clone https://github.com/ojhall94/stateoftheuniverse
cd stateoftheuniverse
python3 setup.py install --user
```

Comment (astromariarosa) [reply README.md:10]: Got it, thanks!

File: README.md [hunk 2]
```diff
@@ -1,13 +1,40 @@
...
-Proposed by Mariarosa Marinelli
+## A python package that returns the current state of the universe
 
-And made happen by:
-- Mariarosa Marinelli[![github icon](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/22/github.png)](https://github.com/astromariarosa/)
-- Ania Kapuścińska[![github icon](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/22/github.png)](https://github.com/lambdanis/)
-- Timothy Gebhard[![github icon](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/22/github.png)](https://github.com/timothygebhard/)
-- Eli Prater[![github icon](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/22/github.png)](https://github.com/pratere/)
-- Oliver Hall[![github icon](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/22/github.png)](https://github.com/ojhall94/)
+Would you like to know when the phase of the moon? What about the current tally of exoplanets and asteroids? Curious about the constellations and planets you can see in the sky tonight? Want to celebrate the birthdays of historical astronomers?
```
Comment (Pratere) [inline README.md:6]: "Would you like to know when the phase of the moon?" Did you mean to say "Would you like to know which phase the moon is in?"

**Review by Pratere — APPROVED** [review]

**Pull request merged**